src/org/lwjgl/opengl/awt/GLData.java

```diff
@@ -120,6 +124,11 @@ public static enum ReleaseBehavior {
      * Whether to use sRGB color space.
      */
     public boolean sRGB;
+    
+    /**
+     * <b>For internal use only</b>, used to determine the sRGB extension to use
+     */
+    public SRGB buffer_sRGB = SRGB.EXT_sRGB;
```

Comment:
I believe this can be a package-private boolean flag, no? The field should be encapsulated rather than the user being told in the documentation that it should not be used.

Reply:
I hadn't thought of it that way (working on it)...

Reply:
@JNightRider just seeing this now (sorry!). What about the public access to this field?

Reply:
Hi @SWinxy 

That was to determine whether to use `WGL_FRAMEBUFFER_SRGB_CAPABLE_EXT` or `WGL_FRAMEBUFFER_SRGB_CAPABLE_ARB`, but I've since changed it to a boolean value that can only be accessed at the package level.

Public access was not correct (as you had told me) and has been modified (it is now updated). If you can review the new changes it would be great.